modules/bridge/src/main/ts/ephox/bridge/components/dialog/DialogFooterButton.ts

```diff
@@ -56,7 +63,14 @@ export interface DialogFooterMenuButton extends BaseDialogFooterButton {
   items: DialogFooterToggleMenuItem[];
 }
 
-export type DialogFooterButton = DialogFooterNormalButton | DialogFooterMenuButton;
+export interface DialogFooterTogglableIconButton extends Omit<BaseDialogFooterButton, 'icon'> {
+  type: 'togglableIconButton';
+  text: string;
+  tooltip: Optional<string>;
```

Comment:
Why do we need optional `tooltip` here? It can be derived from the mandatory text.
Or maybe `text` should be optional as well?

Reply:
since the `text` is not showed anyway I'll remove it and made the `tooltip` mandatory

Reply:
since the text is not showed anyway I'll remove it and made the `tooltip` mandatory

Reply:
sorry looking better `text` is in the base, maybe is worth to use it as a `tooltip` and remove `tooltip` field

Reply:
ok, removed the text in the end